bigquery_etl/metadata/parse_metadata.py

```diff
@@ -428,11 +428,11 @@ def set_bigquery_clustering(self, clustering_fields):
             time_partitioning = self.bigquery.time_partitioning
             range_partitioning = self.bigquery.range_partitioning
 
-        self.bigquery = BigQueryMetadata(
-            time_partitioning=time_partitioning,
-            range_partitioning=range_partitioning,
-            clustering=ClusteringMetadata(fields=clustering_fields),
-        )
+            self.bigquery = BigQueryMetadata(
+                time_partitioning=time_partitioning,
+                range_partitioning=range_partitioning,
+                clustering=ClusteringMetadata(fields=clustering_fields),
+            )
```

Comment:
Are there cases where we might want to set clustering when `bigquery` metadata isn't already defined though?

An alternative solution could be to initialize `time_partitioning` and `range_partitioning` to `None` at the top of the function before the `if self.bigquery:` statement (similar to the logic in `set_bigquery_partitioning()`).

Reply:
> Are there cases where we might want to set clustering when `bigquery` metadata isn't already defined though?

Come to think of it, the errors we were seeing in Airflow would seem to be evidence that that is the case, and this change will result in the clustering not being set for whatever those table(s) were.

Reply:
follow up https://github.com/mozilla/bigquery-etl/pull/5693